lib/web_ui/test/text/measurement_test.dart

```diff
@@ -1154,5 +1154,7 @@ EngineLineMetrics line(
     width: width,
     lineNumber: lineNumber,
     left: left,
+    // TODO(yjbanov): I chose -1 value just to fix the analyzer warning. I don't know if it's the right value to pass.
```

Comment:
The default is -1. No need for TODO. /cc @mdebbar

Reply:
Done.